docs/general/builtins/scalar-functions.rst

```diff
@@ -774,6 +774,90 @@ If you don't specify a time zone, ``truncate`` uses UTC time::
     +---------------+---------------+
     SELECT 3 rows in set (... sec)
 
+.. _date-bin:
+
+``date_bin(interval, timestamp, origin)``
+-----------------------------------------
+
+
+``date_bin`` "bins" the input timestamp to the specified interval, aligned with
+a specified origin.
+
+``interval`` is a expression of type ``interval``. The ``timestamp`` and
+``origin`` parameters are expressions of type ``timestamp with time zone`` or
+``timestamp without time zone``. The return type matches the ``timestamp`` and
+``origin`` parameter types and will be either ``timestamp with time zone`` or
+``timestamp without time zone``.
```

Comment:
I would format this a bit different:
``interval``is an expression of type ``interval``.
``timestamp`` and ``origin`` are expressions if type ``timestamp with time zone`` or ``timestamp without time zon``.
The return type matches the ``timestamp`` and ``origin`` types and will be either ``timestamp with time zone`` or ``timestamp without time zon``.